### ISSUE TITLE
will you please share the hyper-parameter for RTE distill?

### ISSUE THREAD

**Issue by 1024er** [huawei-noah/Pretrained-Language-Model#52]
Hi, I used the default hyper-parameter in TinyBERT repo, and the result on RTE is 30.7 on dev, and 28.6 on test,  much far from the results in the paper. So will you please share the hyper-parameter for RTE distill? Thank you.

**Comment by FFChyan** [comment]
Same here, seems that the default hyper-parameters are not suitable for MRPC and SST-2 tasks ether. It would be much helpful if someone shares the hyper-parameters. Many thanks!

**Comment by FFChyan** [comment]
Solved. I mistakenly use bert-base-uncased as the teacher model, which is a pretrained model instead of a fine-tuned one. When using bert-base-uncased model fine-tuned on a specific task, the accuracy is normal. Hope to be helpful!

> Same here, seems that the default hyper-parameters are not suitable for MRPC and SST-2 tasks either. It would be much helpful if someone shares the hyper-parameters. Many thanks!
